### ISSUE TITLE
Using custom theme fontSize named "paragraph" crashes the Expo App on Android Only

### ISSUE THREAD

**Issue by omaziarz** [nativewind/nativewind#330]
**Describe the bug**

I am using nativewind with Expo and didn't test this in a non Expo RN app.
I have a custom theme where i defined the following: 
```typescript
module.exports = {
  theme: {
    fontSize: {
        paragraph: '17px',
    }
  }
}
```
if i use it like this :
```typescript
<Text className="text-paragraph">Text</Text>
```
in any text component my app will crash without error only on Android. On iOS the app works fine and the style is applied.

Renaming `paragraph` fixes the app.
Firstly i though that maybe paragraph was a reserved keyword in nativewind but it works on iOS

The real problem is that there are no errors shown the app crashes without any feedback. To find the issue of the crashes i had to comment all my components one by one

**To Reproduce**
1. Have an Expo app
2. Use the described tailwind theme
3. On a text component add the following className: "text-paragraph"
4. The Expo app will crash after the splashscreen on Android

**Expected behavior**
The app shouldn't crash on android when i use "text-paragraph"

**Comment by marklawlor** [comment]
Should be fixed with `nativewind@next`. See #308 for more information

**Comment by Jonatthu** [comment]
@marklawlor seems like gap-2, flex-1 and flex-row | flex-col and font sizes like text-9xl are broken on the 22 version

<img width="532" alt="image" src="https://user-images.githubusercontent.com/3742176/202917888-18f7a696-a16a-40da-bb54-ec71efc7d9c5.png">

<img width="403" alt="image" src="https://user-images.githubusercontent.com/3742176/202917901-34f82dd0-0896-44b2-b91f-4f13659de5a1.png">



This is the code for the buttons that renders good on web but renders broken on iOS.

```
			<View className="mb-4 gap-2">
				{DecimalsButtons.map((row, index) => (
					<View className="flex-1 flex-row gap-2" key={index}>
						{row.map((button) => (
							<Button
								className="h-16 flex-1"
								key={button}
								size="giant"
								testID={DecimalPadLocators.Decimal}
								onPress={() => onPress(button)}
							>
								{button}
							</Button>
						))}
					</View>
				))}
			</View>
```

Font sizes like text-9xl are broken, this would render good on web but broken on iOS, if I change the class name to text-base that would work again but with different size of course.

```
						<Text
							className="text-9xl"
							status="primary"
						>
							{FormatToDollars(state)}
						</Text>
```

**Comment by marklawlor** [comment]
@Jonatthu can you please share your repo? At this stage the problems are not "this class doesn't work" rather "this project setup breaks the classes from compiling"

**Comment by Jonatthu** [comment]
@marklawlor https://github.com/Jonatthu/expo-template-storybook/blob/skia-setup/App.tsx
This is a reproduction of the issue, text-9xl, 8xl and others are broken and also flex is nor working as expected, on my example it supposed to create a 3x3 column but looks like renders everything in a single row without no height

**Comment by marklawlor** [comment]
Fixed in `3.0.0-next.23`.

**Comment by danstepanov** [comment]
We are closing all pre-v4.1 issues now that v4.1 is released. If the problem is still happening with the latest v4.1, [please open a new issue](https://github.com/nativewind/nativewind/blob/next/contributing.md#opening-an-issue) and reference this one in the description.
